$MirOS: ports/devel/autoconf/2.61/patches/patch-lib_autoconf_autotest_m4,v 1.1 2007/05/08 20:38:26 tg Exp $
--- lib/autoconf/autotest.m4.orig	Sat Apr  5 23:04:25 2008
+++ lib/autoconf/autotest.m4	Sat Nov 21 23:28:30 2009
@@ -65,7 +65,7 @@ AC_DEFUN([AC_CONFIG_TESTDIR],
 [AC_CONFIG_COMMANDS([$1/atconfig],
 [cat >$1/atconfig <<ATEOF
 @%:@ Configurable variable values for building test suites.
-@%:@ Generated by $[0].
+@%:@ Generated by $[0] @VERSION@.
 @%:@ Copyright (C) 2000, 2001, 2003, 2004 Free Software Foundation, Inc.
 
 # The test suite will define top_srcdir=$at_top_srcdir/../.. etc.
